doc/vim-lsp.txt

```diff
@@ -603,6 +603,25 @@ g:lsp_completion_resolve_timeout         *g:lsp_completion_resolve_timeout*
     The `completionItem/resolve` request's timeout value.
     If your vim freeze at `CompleteDone`, you can set this value to 0.
 
+g:lsp_custom_open_location_fn				*g:lsp_custom_open_location_fn*
```

Comment:
I don't know the vim-lsp's option naming convention.
But I think `_fn` suffix is not needed here because another option has no suffix.

And Type is `List` now. Could you update docs?

Reply:
And sorry, please add default definition in `plugin/lsp.vim`.

Reply:
All done.